server/datastore/psql/delete_maintenance_request.go

```diff
@@ -0,0 +1,10 @@
+package psql
+
+
```

Comment:
Can you add a comment with a brief description of the function?

Reference: https://blog.golang.org/godoc